alphadia/constants/default.yaml

```diff
@@ -53,7 +53,7 @@ library_prediction:
 
 # define custom alphabase modifications not part of unimod or alphabase
 # also used for decoy channels
-custom_modififcations:
```

Comment:
It’s breaking in theory :D alphabase can’t use the custom mods with multiprocessing yet. So it’s not being used (yet)